### ISSUE TITLE
Regarding class weights in bce loss function

### ISSUE THREAD

**Issue by puneeshkhanna** [dungnb1333/SIIM-COVID19-Detection#7]
weight=torch.FloatTensor([0.2, 0.2, 0.3, 0.3]

Above parameter is corresponding to inputs in a batch.

To handle class imbalance, I think the parameter is pos_weight and not weight. 

https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html?highlight=bcewithlogitsloss#torch.nn.BCEWithLogitsLoss
